### ISSUE TITLE
Bypass block for job composer

### ISSUE THREAD

**Issue by ashton22305** [OSC/ondemand#4959]
The job composer does not have the bypass block present in the rest of OOD. This violates [WCAG SC 2.4.1 Bypass Blocks](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip)

**Comment by Bubballoo3** [comment]
Just to clarify, this refers to the 'Skip Navigation' button, and should be pretty easy to add in
